### PR TITLE
feat: add oras pkg

### PR DESCRIPTION
**Pull request by tuananh**

### CODE REVIEW

**Review by developer-guy — DISMISSED** [review]

File: oras.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/oras-project/oras
+      branch: v${{package.version}}
+      destination: oras
+  - runs: |
+      cd oras
+      make build-linux-amd64
+      make build-linux-arm64
```
Comment (kaniini) [inline oras.yaml:30]: hmm?  why not use subpipelines here?

e.g.

```yaml
- assertions:
    required-steps: 1
  pipeline:
    - if: ${{build.arch}} == 'x86_64'
      runs: |
        make build-linux-amd64
    - if: ${{build.arch}} == 'aarch64'
      runs: |
        make build-linux-arm64
- runs: |
    install -m755 -D ./bin/linux/*/oras "${{targets.destdir}}"/usr/bin/oras
```

Comment (tuananh) [reply oras.yaml:30]: Nice. Let me change it.

File: oras.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package:
+  name: oras
+  version: 0.16.0
```
Comment (FeynmanZhou) [inline oras.yaml:3]: [ORAS CLI v1.0.0-rc.1](https://github.com/oras-project/oras/releases/tag/v1.0.0-rc.1) is the latest version. Do you consider using it here?

**Review by FeynmanZhou — COMMENTED** [review]
Thanks for adopting ORAS in Wolfi, just a small question : )

File: oras.yaml (hunk 2 above)
Comment (tuananh) [reply oras.yaml:3]: I guess I can change to rc release if no one oppose to this

Comment (developer-guy) [reply oras.yaml]: I assume, you can go for it @tuananh

File: oras.yaml (hunk 1 above)
Comment (developer-guy) [reply oras.yaml]: cool!

File: oras.yaml (hunk 2 above)
Comment (rawlingsj) [reply oras.yaml]: > [ORAS CLI v1.0.0-rc.1](https://github.com/oras-project/oras/releases/tag/v1.0.0-rc.1) is the latest version. Do you consider using it here?

Though this a pre release?  I wonder if we should keep to released versions.

Comment (tuananh) [reply oras.yaml:3]: Lets do 0.16.0 and wait for 1.0.0

File: oras.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/oras-project/oras
+      branch: v${{package.version}}
+      destination: oras
```
Comment (imjasonh) [inline oras.yaml:26]: Can we use `fetch` here, or `expected-commit` that [someone very smartly added recently](https://github.com/chainguard-dev/melange/pull/275) 😄

Comment (tuananh) [reply oras.yaml:26]: oras uses git commit in ldflags in their Makefile. that's why i use git-checkout instead of fetch.

**Review by imjasonh — APPROVED** [review]

**Pull request merged**